pkg/server/server.go

```diff
@@ -212,6 +214,14 @@ func (s *Server) CreateCertificate(ctx context.Context, icr *securityapi.IstioCe
 		duration = s.opts.MaximumClientCertificateDuration
 	}
 
+	// If custom client duration is specified, override with the value.
+
+	if s.opts.ClientCertificateDuration > 0 {
+		duration = s.opts.ClientCertificateDuration
+	}
```

Comment:
I get a bit confused by this code that may modify `duration` twice (max and override)., and wonder if a shared switch statement would make this clearer.